include/simulation/SimulatorManager.hpp

```diff
@@ -51,11 +51,11 @@ class SimulatorManager {
     SimulatorManager(SimulatorNetworkConfiguration configuration);
     ~SimulatorManager();
 
-    // Both of the send functions return amount of bytes sent, return -1 if error occured
+    // Both of the send functions return amount of bytes sent, return -1 if error occurred
     int sendRobotControlCommand(RobotControlCommand& robotControlCommand, rtt::Team color);
```

Comment:
Doesn't it return 0?

Reply:
Yeah, so there was an inconsistency between send functions. The networkers within RTT return 0 on error, but the socket that sends to the Simulator still returns -1 on errors. In the next commits I changed this to returning 0 on errors too for consistency.